### DISCUSSION TITLE
Making a single page landscape in a portrait document

### DISCUSSION THREAD

**Discussion by newzealandpaul** [quarto-dev/quarto-cli#2791] [Q&A] [answered]
I am authoring a PDF report with Quarto (papersize: a4, documentclass: article) and would like to know the best way to make a single page (with a mermaid diagram) portrait.

I am currently archiving it with

> \newpage
> \KOMAoptions{paper=landscape,pagesize}
> \`\`\`{mermaid}
> ...
> \`\`\`
> \newpage
> \KOMAoptions{paper=portrait,pagesize}
> 

But this has two problems.

The first is that I lose page numbers on the landscape page.

The second problem is that it removes/overwrites the YAML header's geometry settings for the entire document.  

I had to resort to adding the following in my header.tex

> \usepackage[top=20mm,left=30mm,heightrounded ]{geometry}

Is there a better way of making a single page landscape in a portrait article? Or is this something I should make a feature request for.

**Comment by dragonstyle** [comment]
I don't have any first hand experience with this though a couple of ideas:

1) The geometry package does support `\newgeometry` and `\restoregeometry` commands which could allow you to alter page geometric for a single page and then restore the old geometry.

2) You could experiment with packages like pdflscape package, which creates an environment that emits landscape pages... (https://ctan.math.washington.edu/tex-archive/macros/latex/contrib/pdflscape/pdflscape.pdf)

**Comment by mcanouil** [comment]
Related:

- https://github.com/quarto-dev/quarto-cli/discussions/6104
- https://github.com/quarto-dev/quarto-cli/discussions/680
- https://github.com/quarto-dev/quarto-cli/issues/8512

**Comment by Fpadt** [comment, marked as answer]
I just installed the quarto pre-release 1.6.23 and landscape come out-of-the-box, see: [https://github.com/quarto-dev/quarto-cli/pull/10581](https://github.com/quarto-dev/quarto-cli/pull/10581),  
else you can use the extension from which it originates [edvinsyk/quarto-landscape](https://github.com/edvinsyk/quarto-landscape)]

**Reply by mcanouil** [reply]
See https://prerelease.quarto.org/docs/authoring/article-layout.html#landscape-mode

(upon release on https://quarto.org/docs/authoring/article-layout.html#landscape-mode)

**Reply by newzealandpaul** [reply]
Thank you that solves the issue
